pkg/cmd/pulumi/convert/convert.go

```diff
@@ -558,6 +558,7 @@ func generateAndLinkSdksForPackages(
 			pkg.Name,
 			&plugin.ParameterizeValue{Value: pkg.Parameterization.Value},
 			registry,
+			env.Global(),
```

Comment:
Passing this env around could be a simple PR to pull out to get in over Christmas.